packages/itwinui-css/src/menu/mixins.scss

```
@@ -51,7 +51,7 @@ $iui-active-outline: 1px solid var(--iui-color-border-accent);
     margin-block-start: -1px;
   }
 
-  &:where([data-iui-actionable='true'], .iui-menu-item):where(:hover),
+  &:where([data-iui-actionable='true'], .iui-list-item):where(:hover),
```

I believe this was just leftover from #1295 so it should be removed altogether.

```suggestion
  &:where([data-iui-actionable='true']:hover),
```

By default, `iui-list-item` should *not* have a hover state. See [docs](https://itwinui.bentley.com/docs/list#actionable).

To fix the `menu.html` page, `data-iui-actionable="true"` should be added to all instances of `<div class="iui-list-item">`. See [React implementation](https://github.com/iTwin/iTwinUI/blob/e1045a8ea4e2d36f4b67342e50dfb8278c264217/packages/itwinui-react/src/core/Menu/MenuItem.tsx#L148-L152) for comparison.

---

Ah, this makes sense.  Committed your suggestion and updated the CSS demo page to use `data-iui-actionable='true'`.